pep-9999.rst

```diff
@@ -0,0 +1,136 @@
+PEP: 9999
+Title: Remove wstr from Unicode
+Author: Inada Naoki <songofacandy@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 24-Jun-2020
+Python-Version: 3.10
+
+
+Abstract
+========
+
+PEP 393 deprecated some unicode APIs, and introduced ``wchar_t *wstr``,
+and ``Py_ssize_t wstr_length`` in unicode implementation to support
+these deprecated APIs. [1]_
+
+This PEP is planning removal of ``wstr``, and ``wstr_length`` with
+deprecated APIs using these members by Python 3.12.
+
+
+Motivation
+==========
```

Comment:
Can you try to explain a bit the history of the PEP 393? Mention that it's implemented in Python 3.3, it was created to reduce the memory footprint and better support the full Unicode Character Set (previously, Python was built with UCS2 on all platforms but Windows which used UCS4).

For me, the important part is to give the Python version 3.3 and its release year (2012), to put in perspective the deprecation and the removal.

I consider that all these functions are deprecated since 2012, even if there were only deprecated in documentation after Python 3.3 and Py_DEPRECATED() was only added "recently".

Python 3.12 should be released at the end of 2023: it would means that these functions were deprecated for 11 years! Knowing that, I think that it's perfectly reasonable to schedule the removal in Python 3.12.

Maybe also remind that there was a policy (I forgot which PEP) to keep these functions around for compatibility with Python 2, but Python 2 support stopped in 2020.